custom/icds_reports/utils/state_level.sql

```diff
@@ -0,0 +1,32 @@
+SELECT
```

Comment:
It is obviously too late to change this now but why wasn't this already accessible in our existing agg awc daily view that is used elsewhere in the dashboard? https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/migrations/sql_templates/database_views/agg_awc_daily.sql
@ShubhamBansal1997